apps/web/pages/api/cron/monitorDailyWebhooks.ts

```diff
@@ -0,0 +1,35 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { getWebhooks } from "@calcom/core/videoClient";
+import { sendSendgridMail } from "@calcom/features/ee/workflows/lib/reminders/providers/sendgridProvider";
+import { SENDER_NAME } from "@calcom/lib/constants";
+import { defaultHandler } from "@calcom/lib/server";
+
+async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const apiKey = req.headers.authorization || req.query.apiKey;
+  if (process.env.CRON_API_KEY !== apiKey) {
+    res.status(401).json({ message: "Not authenticated" });
+    return;
+  }
+
+  const webhooks = await getWebhooks();
+
+  const inactiveWebhooks = webhooks.filter((webhook) => webhook.state !== "ACTIVE");
+  if (inactiveWebhooks.length) {
+    // Notify by sending email
+    await sendSendgridMail(
+      {
+        to: process.env.ALERT_EMAIL_TO,
+        subject: "[HIGH] Alert: Daily Webhooks are not active",
+        html: `<body style="white-space: pre-wrap;">Daily Webhook url api/recorded-daily-video is not functional.</body>`,
```

Comment:
This is just communicating the problem. We should also inform about the possible ways to fix the problem

Reply:
Actually we have debug this issue ourselves. there is no info about the error

Reply:
more info here https://github.com/calcom/cal.com/pull/15126#issuecomment-2167439247